md/single-sign-on-with-saml.md

```diff
@@ -247,22 +247,33 @@ com.bonitasoft.level = ALL
 
 ## Configure logout behaviour
 
-#### Bonita Portal
-
 When your Bonita platform is configured to manage authentication over SAML, when users log out of Bonita Portal, they do not log out of the SAML Identity Provider (IdP).  
 Therefore they are not logged out of all applications that are using the IdP.  
-To avoid this, you can hide the logout option of the portal.  
-To do this, set the `logout.link.hidden=true` option in `authenticationManager-config.properties` located in `platform_conf/initial/tenant_template_portal` 
-for not initialized platform or `platform_conf/current/tenant_template_portal` and `platform_conf/current/tenants/[TENANT_ID]/tenant_portal/`.
+To avoid this, you have two options :  
+
+#### Hide the logout button of the portal
+
+This is the most commonly used solution.  
+To do this, set the `logout.link.hidden` option to `true` in `authenticationManager-config.properties` located in `platform_conf/initial/tenant_template_portal` for not initialized platform or `platform_conf/current/tenant_template_portal` and `platform_conf/current/tenants/[TENANT_ID]/tenant_portal/`.
 
 ::: info
 **Note:** When a user logs out from the IdP, Bonita Portal's session will remain active. The user's session time to live will be reset 
```

Comment:
How can Andy log out from the IdP if no "Logout" button is provided? 
(Do not pay attention to my comment if this seems so obvious.)

Reply:
IdP usually offer a page allowing at least to log out in addition of their login page